web/src/features/charts/hooks/useBreakdownChartData.ts

```diff
@@ -130,13 +131,11 @@ function getStorageValue(
     return Number.NaN;
   }
 
-  let scaledValue = (-1 * Math.min(0, storageValue)) / valueFactor;
+  const invertedValue = -1 * Math.min(0, storageValue);
 
-  if (displayByEmissions) {
-    scaledValue *= value.dischargeCo2Intensities[storageKey] / 1e3 / 60;
-  }
-
-  return scaledValue;
+  return displayByEmissions
+    ? invertedValue * value.dischargeCo2Intensities[storageKey] * valueFactor
+    : invertedValue / valueFactor;
```

Comment:
With all this multiplication or division there is a risk we introduce floating point errors.

Are we sure these are rounded upstream?

Reply:
It's not ideal, but we'll attempt to move most of these calculation to app-backend in follow-up work :)